src/DoocsBackend.cc

```diff
@@ -176,14 +178,14 @@ namespace ChimeraTK {
       // if again error received, throw exception
       if(rc && isCommunicationError(dst.error())) {
         lk.unlock();
-        setException();
-        throw ChimeraTK::runtime_error(std::string("Cannot read from DOOCS property: ") + dst.get_string());
+        auto message = std::string("Cannot read from DOOCS property: ") + dst.get_string();
```

Comment:
Are you sure that dst.get_string() is giving the address/name of the property? Isn't this supposed to give the data content as a string? This did not change, though, so it's OK for this merge request.

Reply:
I am pretty sure it works, as I likely have added this in need for this particular information.